src/angular/notification-toast/notification-toast-container.scss

```diff
@@ -59,9 +51,9 @@
 }
 
 .sbb-notification-toast-icon-wrapper {
-  min-width: pxToRem(24);
-  width: pxToRem(24);
-  height: pxToRem(24);
+  min-width: var(--sbb-icon-size-default);
+  width: var(--sbb-icon-size-default);
+  height: var(--sbb-icon-size-default);
   margin-left: pxToRem(16);
   border: none;
   background: none;
```

Comment:
should we not rewrite .sbb-notification-toast-mobile? and move .sbb-notification-toast mobile definition to .sbb-notification-toast ?

Reply:
(you don't see it in the above code snippet)